lib/mayaUsd/utils/editability.cpp

```diff
@@ -0,0 +1,52 @@
+//
+// Copyright 2021 Autodesk
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+#include "editability.h"
+
+PXR_NAMESPACE_OPEN_SCOPE
+
+/*! \brief  The tokens used in the lock metadata.
+ */
+TfToken Editability::MetadataToken("mayaLock");
```

Comment:
It's time to consolidate all the metadata token into a single file. Similar to what's was done with optionvars - https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/base/tokens.h#L32. Please create another set of tokens in that file and consolidate selectability, editability, [custom layer data](https://github.com/Autodesk/maya-usd/blob/dev/lib/mayaUsd/utils/customLayerData.h#L30), and look for others in the mayaUsd library.

Reply:
Oh right, I forgot that, will do.